modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/subcellularmodel/v0.1.2.json

```diff
@@ -0,0 +1,37 @@
+{
+  "@context": [
+    "{{base}}/contexts/neurosciencegraph/core/schema/v0.1.0",
+    {
+      "this": "{{base}}/schemas/neurosciencegraph/simulation/subcellularmodel/v0.1.2/shapes/"
+    }
+  ],
+  "@type": "nxv:Schema",
+  "imports": [
+    "{{base}}/schemas/neurosciencegraph/commons/modelinstance/v0.1.1"
+  ],
+  "shapes": [
+    {
+      "@id": "this:SubCellularModelShape",
```

Comment:
There is no SynapseRelease to point to ?

Reply:
The release entity will contain the collection of the models.

Reply:
How many models do  you expect to be part of a SynapseRelease ? The main reason why isPartOf was used was to avoid a collection of hundreds of thousands of model's uris in a SynapseRelease payload.

Reply:
Ok, we are keeping isPartOf.